Project.toml

```diff
@@ -0,0 +1,17 @@
+name = "LibPNG"
+uuid = "108bb7d8-ee5e-11e9-2f8f-6ba5bcecef1c"
+authors = ["msb <mandeep.bahal@roames.com.au>"]
+version = "0.1.0"
+
+[deps]
+BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+MicroLogging = "cecf6091-0e4e-54ba-8cba-f0e96a51f282"
```

Comment:
You may be able to remove MicroLogging if you're now using stdlib Logging instead?  (Also it might be an optional dependency if it's only used in the tests?)

Reply:
Done